frontend/awx/resources/sources/InventorySourceSubForm.tsx

```diff
@@ -42,15 +42,23 @@ export function InventorySourceSubForm() {
           <PageFormCredentialSelect<InventorySourceForm>
             name="credential"
             label={t('Credential')}
-            placeholder={t('Add credential')}
-            labelHelpTitle={t('Credential')}
             labelHelp={t(
               'Select credentials for accessing the nodes this job will be ran against. You can only select one credential of each type. For machine credentials (SSH), checking "Prompt on launch" without selecting credentials will require you to select a machine credential at run time. If you select credentials and check "Prompt on launch", the selected credential(s) become the defaults that can be updated at run time.'
             )}
-            isRequired={sourceTypes.slice(1).includes(source as string)}
-            credentialIdPath="credentialIdPath"
-            sourceType={source as string}
-            acceptableCredentialKinds={['kubernetes', 'cloud']}
```

Comment:
This was not working if it is single select, but I believe do want this to filter. We need to double check.

Reply:
https://github.com/ansible/ansible-ui/pull/2455/commits/89358f75e363c99805ab6fc3f606d6c7a22e1053 Added kubernetes in follow-up commit